integration-tests/src/tests/network/runner.rs

```diff
@@ -574,9 +498,6 @@ impl Runner {
                 setup_network_node(account_id, validators, chain_genesis, network_config)
             })
             .await,
-            events: recv_events,
-            pings: MultiSet::default(),
```

Comment:
that's great that we can remove those :). In case you didn't do that yet, please check whether broadcast::unbounded_channel() above can also be already removed altogether.